telegram/ext/jobqueue.py

```diff
@@ -376,7 +377,8 @@ class Job(object):
             Only optional for backward compatibility with ``JobQueue.put()``.
         tzinfo (:obj:`datetime.tzinfo`, optional): timezone associated to this job. Used when
             checking the day of the week to determine whether a job should run (only relevant when
-            ``days is not Days.EVERY_DAY``). Defaults to UTC.
+            ``days is not Days.EVERY_DAY``); also used when :attr:`next_t` is retrieved.
```

Comment:
Personally I'd think this goes without saying. Can be removed IMO.

Reply:
Sure, fixed.